proto/api/v1/project.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package api.v1;
+
+import "api/v1/common.proto";
+
+// ProjectService serves project related functions
+service ProjectService {
+  // List all accessible projects
+  rpc List(ProjectServiceListRequest) returns (ProjectServiceListResponse) {
+    option (visibility) = VISIBILITY_SELF;
+  }
```

Comment:
I think `Get` should also be added for the sake of completeness, e.g. when the frontend wants to retrieve project information just for the active project or something.

```
  rpc Get(ProjectServiceGetRequest) returns (ProjectServiceGetResponse) {
    option (project_roles) = PROJECT_ROLE_OWNER;
    option (project_roles) = PROJECT_ROLE_EDITOR;
    option (project_roles) = PROJECT_ROLE_VIEWER;
    option (auditing) = AUDITING_EXCLUDED;
  }
```

Reply:
done